core/src/test/java/com/redhat/lightblue/client/response/TestDefaultLightblueBulkDataResponse.java

```diff
@@ -53,7 +53,7 @@ private DefaultLightblueBulkDataResponse simpleSetUp() throws Exception {
         bulkRequest.add(dfr);
         bulkRequest.add(dfr2);
 
-        return new DefaultLightblueBulkDataResponse(jsonResponse, bulkRequest);
+        return new DefaultLightblueBulkDataResponse(jsonResponse, null, bulkRequest);
```

Comment:
Maybe keep the DefaultLightblueBulkDataResponse(jsonResponse, bulkRequest) constructor for the cases where there are no headers?